design-proposals/volume-update-strategy.md

```diff
@@ -0,0 +1,216 @@
+# Overview
```

Comment:
You should mention somewhere that this feature relies on the `vmRolloutStrategy` field set to `LiveUpdate`, which also needs the `VMLiveUpdateFeatures` feature gate to be enabled.

Reply:
Here, I'm a bit unsure how to proceed. We introduce already the new field `updateVolumesStrategy`. Do we want also to force the user to set the `vmRolloutStrategy`? I would also use a separate feature gate

Reply:
Clarify this offline and added on the proposal